client/me/notification-settings/comment-settings/index.jsx

```diff
@@ -68,29 +59,35 @@ const NotificationCommentsSettings = React.createClass( {
 		}
 
 		return ( <p className="notification-settings-comment-settings__placeholder">&nbsp;</p> );
-	},
+	};
 
 	render() {
+		const {
+			path,
+			translate
+		} = this.props;
+
 		return (
 			<Main>
+				<QueryUserDevices />
 				<MeSidebarNavigation />
 				<ReauthRequired twoStepAuthorization={ twoStepAuthorization } />
 
-				<Navigation path={ this.props.path } />
+				<Navigation { ...{ path } } />
```

Comment:
No, just excessive refactoring when _es6fiying_ this component. Removed on 6b565c5df348c61222fe569f5a8ce719c4731f95.